go/pserver/cclient/cclient.go

```diff
@@ -100,9 +105,39 @@ func paddle_new_pserver_client(addrs *C.char, selected int) C.paddle_pserver_cli
 }
 
 //export paddle_new_etcd_pserver_client
-func paddle_new_etcd_pserver_client(etcd_addr *C.char) C.paddle_pserver_client {
-	// TODO(helin): fault tolerant pserver client using etcd.
-	panic("not implemented.")
+func paddle_new_etcd_pserver_client(etcd_addr *C.char, pserver_path *C.char, selected int) C.paddle_pserver_client {
+	if etcd_addr == "" {
+		return C.PSERVER_ERROR
+	}
+	etcd_addrs := strings.Split(etcd_addr, ",")
+	cli, err := clientv3.New(clientv3.Config{
+		Endpoints:   etcd_addrs,
+		DialTimeout: etcdTimeout,
+	})
+	if err != nil {
+		log.Errorln(err)
+		return err
```

Comment:
The return type is `C.paddle_pserver_client`, I think the function would panic if client connect etcd failed.